packages/superset-ui-plugin-chart-table/src/Table.tsx

```diff
@@ -60,16 +67,20 @@ function getCellHash(cell: Cell) {
   return `${cell.key}#${cell.value}`;
 }
 
-function getText(value: unknown) {
+function getText(value: unknown, format: TimeFormatter | NumberFormatter | undefined) {
+  let formattedValue = value;
+  if (format) {
+    formattedValue = format.format(value as any);
```

Comment:
Should we return here? If there is a `format` function, that should not produce html output.